stdlib/nodejs/pathname.rb

```diff
@@ -0,0 +1,31 @@
+class Pathname
+  include Comparable
+
+  @__path__ = `require('path')`
+  `var __path__ = #{@__path__}`
+
+  def initialize(path)
+    if Pathname === path
+      @path = path.path.to_s
+    elsif path.respond_to?(:to_path)
+      @path = path.to_path
+    elsif path.is_a?(String)
+      @path = path
+    elsif path.nil?
+      raise TypeError, 'no implicit conversion of nil into String'
+    else
+      raise TypeError, "no implicit conversion of #{path.class} into String"
+    end
+    raise ArgumentError if @path == "\0"
+  end
+
+  attr_reader :path
+
+  def absolute?
+    `__path__.isAbsolute(#{@path.to_str})`
```

Comment:
Use `isAbsolute` otherwise `c:/absolute/path` is not considered as an absolute path by the `stdlib/pathname` implementation.